canmatrix/canmatrix.py

```diff
@@ -198,17 +202,17 @@ def multiplex(value):
                 multiplex = value
             return multiplex
 
-        float_factory = kwargs.pop('float_factory', float)
+        self.float_factory = kwargs.pop('float_factory', float)
```

Comment:
As a nitpick, I think the `self` part could be dropped to tidy the diff.

Reply:
fixed